src/network/reducer.es

```diff
@@ -1,11 +1,11 @@
 export default (state = {}, action) => {
-  if (action.meta && typeof action.meta.isNetworkActive === 'boolean') {
-    const isNetworkActive = action.meta.isNetworkActive;
-    let activeCount = state.activeCount || 0;
-    if (isNetworkActive) activeCount += 1;
-    else activeCount -= 1;
+  if (action.meta && typeof action.meta.isRequestActive === 'boolean') {
+    const diff = action.meta.isRequestActive ? 1 : -1;
 
-    return { activeCount, isNetworkActive };
+    const activeRequestsCount = Math.max((state.activeRequestsCount || 0) + diff, 0);
```

Comment:
by using `Math.max` `activeRequestsCount` should never be less than 0

Reply:
Correct, negative requests make no sense. If it happens something is broken.